kerastuner/tuners/bayesian.py

```diff
@@ -1,16 +1,105 @@
+import math
 import random
 
 import numpy as np
+from scipy import linalg
```

Comment:
scipy is only used in this file -- in any ideal world we'd get rid of it (the math & optimization in this file can be implemented in TF, and it would likely end up being faster). However, moving from depending on both scipy and sklearn to only depending on scipy is a strict improvement, so we can merge this PR.

Reply:
Thank you for the comment! I will continue to work on removing scipy from dependency.